renderer.go

```diff
@@ -15,6 +16,7 @@ type Renderer interface {
 	Add(name string, tmpl *template.Template)
 	AddFromFiles(name string, files ...string) *template.Template
 	AddFromGlob(name, glob string) *template.Template
+	AddFromFS(name string, fsys fs.FS, files ...string) *template.Template
```

Comment:
Would be nice to also have an `AddFromFSFuncs` if possible.

Reply:
It's been a while since I created this PR and the maintainers don't seem to care for a merge. So I'll wait for it to be merged before putting any more effort into this.

Reply:
Yea I see, that's a shame. But I used the workaround in #30, not the worst thing in the world 😭